cli/list_users.go

```diff
@@ -0,0 +1,49 @@
+// Copyright © 2020 Solus Project
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"github.com/DataDrake/cli-ng/cmd"
+	log "github.com/DataDrake/waterlog"
+	"github.com/DataDrake/waterlog/level"
+	"github.com/getsolus/qol-assist/core"
+	"strings"
+)
+
+var listUsers = &cmd.CMD{
+	Name:  "list-users",
+	Short: "List users on the system and their associated groups",
+	Alias: "l",
+	Args:  &ListArgs{},
+	Run: func(root *cmd.RootCMD, command *cmd.CMD) {
```

Comment:
Usually, I use a named function rather than an anonymous one with the Run functions. This resets the indentation level and makes the code easier to read when things get heavily nested.